idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -6,9 +6,9 @@ export default `
         <div class="details-header">
           <h1 data-hook="title"></h1>
           <div class="idea-controls">
-            <button data-hook="feedback" id="feedback-button" class="button primary">Give Feedback</button>
-            <button data-hook="uninstall" id="uninstall-button" class="button neutral"><span class="state-change-inner"></span><span class="transition-text">Disabling...</span><span class="default-text">Disable <span data-hook="title"></span></span></button>
-            <button data-hook="install" id="install-button" class="button primary"><span class="state-change-inner"></span><span class="transition-text">Enabling...</span><span class="default-text">Enable <span data-hook="title"></span></span></button>
+            <button data-l10n-id="giveFeedback" data-hook="feedback" id="feedback-button" class="button primary">Give Feedback</button>
+            <button data-hook="uninstall" id="uninstall-button" class="button neutral"><span class="state-change-inner"></span><span data-l10n-id="disableExperimentTransition" class="transition-text">Disabling...</span><span data-l10n-id="disableExperiment" class="default-text">Disable <span data-hook="title"></span></span></button>
```

Comment:
OK, I found the [old PR](https://github.com/mozilla-b2g/gaia/pull/18392) I referred to above.

In order to localize this bit of html:
```html
<p>By proceeding, I agree to the <a href="">terms</a> and 
<a href="">privacy policy</a> of Firefox Cloud Services.</p>
```

I wound up doing this:

properties file:
```
fxa-notice = By proceeding, I agree to the {{tos}} and {{pn}} of Firefox cloud services.
# LOCALIZATION NOTE(fxa-tos): This string corresponds to the {{tos}} placeholder
# in fxa-notice.
fxa-tos = Terms of Service
# LOCALIZATION NOTE(fxa-pn): This string corresponds to the {{pn}} placeholder
# in fxa-notice.
fxa-pn = Privacy Notice
```

in the view rendering method:
```js
// translate outer text
var localize = navigator.mozL10n.localize;
var noticeText = _('fxa-notice');

// string replace to insert inner nodes
var tosReplaced = noticeText.replace(
  '{{tos}}',
  '<a id="fxa-terms" href="#" class="disabled">Terms of Service</a>'
);
var tosPnReplaced = tosReplaced.replace(
  '{{pn}}',
  '<a id="fxa-privacy" href="#" class="disabled">Privacy Notice</a>'
);

// for each level of nesting, we had to render to the DOM
// to have an element to pass to `localize` :-(
this.fxaNotice.innerHTML = tosPnReplaced;

// grab refs to inner els and localize them
this.fxaPrivacy = document.getElementById('fxa-privacy');
localize(this.fxaPrivacy, 'fxa-pn');
this.fxaTerms = document.getElementById('fxa-terms');
localize(this.fxaTerms, 'fxa-tos');
```

Hopefully the new tool handles this common case in a nicer way, looking forward to hearing from the l20n crew ^_^